solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -58,6 +58,7 @@ function update_solution_config(){
     pushd $SCRIPT_LOCATION/k8_cortx_cloud
         echo > solution.yaml
         yq e -i '.solution.namespace = "default"' solution.yaml
+        yq e -i '.solution.deployment_type = "standard"' solution.yaml
         
         yq e -i '.solution.secrets.name = "cortx-secret"' solution.yaml
         yq e -i '.solution.secrets.content.openldap_admin_secret = "seagate1"' solution.yaml
```

Comment:
Seems we need to remove `openldap_admin_secret` as its not present at https://github.com/Seagate/cortx-k8s/blob/v0.2.0/k8_cortx_cloud/solution.yaml#L7

Reply:
@shailesh-vaidya, removed and tested